tests/testCModel.py

```diff
@@ -98,29 +98,36 @@ def tearDown(self):
     def testNoNoise(self):
         """Test that CModelAlgorithm.apply() works when applied to a postage-stamp
         containing only a point source with no noise.
+
+        We still have to pretend there is noise (i.e. have nonzero values in
+        the variance plane) to allow it to compute a likelihood, though.
```

Comment:
Indirectly, yes.  Before this change `CModel_fluxSigma` was exactly zero in this test, which was easy to verify.  Now it's some nonzero number that's hard to estimate in the unit test in a non-circular way, so we have a fuzzy comparison instead.